docs/specifications/tests/Basic-TP/basic01.md

```diff
@@ -1,199 +1,360 @@
+    3. Else, if the *Delegation Found* set is empty, then do
+       1. For each parent zone name in the *AA SOA Found* set do if *Child Zone*
+          is not a [Direct Subdomain] to parent zone name then output
+          *[B01_PARENT_FOUND]*, parent zone name and the set of name server IP
+          addresses for that name.
+       2. If not all parent zone name in *AA SOA Found* are not equal then output
+          *[B01_PARENT_INDETERMINED]* and the whole set of name server IP
+          addresses from the *AA SOA Found* set,
+
+11. If both of the *Delegation Found* and the *AA SOA Found* sets are empty, then
+    do:
+    1. If *Test Type* is "normal test" then do:
+       1. Create "Superdomain" as the domain just above *Child Zone*.
+       2. Output *[B01_NO_CHILD]* with *Child zone* and *Superdomain*.
+    2. If *Test Type* is "[undelegated test]", output *[B01_CHILD_NOT_EXIST]*
+       with *Child Zone*.
+
+12. If the *AA DNAME Found* set is non-empty then do:
+    1. For each DNAME target in the set output *[B01_CHILD_IS_ALIAS]* with name
+       server IP list, *Child Zone* and the DNAME target.
+    2. If not all members of the set have the same DNAME target, output
+       *[B01_INCONSISTENT_ALIAS]* with *Child Zone*.
+
+13. If the *Non-AA Non-Delegation Found* set is non-empty then for each parent
+    zone name from the set do:
+    1. Output *[B01_UNEXPECTED_NS_RESPONSE]* with parent zone name, *Child Zone*
+       and list of name server IP addresses.
+
+
+## Outcome(s)
+
+The outcome of this Test Case is "fail" if there is at least one message with
+the severity level *[ERROR]* or *[CRITICAL]*.
+
+The outcome of this Test Case is "warning" if there is at least one message
+with the severity level *[WARNING]*, but no message with severity level
+*ERROR* or *CRITICAL*.
+
+In other cases, no message or only messages with severity level *[INFO]* or
+*[NOTICE]*, the outcome of this Test Case is "pass".
+
+
+## Special procedural requirements
+
+If either IPv4 or IPv6 transport is disabled, skip [Sending][Send]
+queries over that transport protocol. A message will be outputted reporting that
+the transport protocol has been skipped.
+
+The *Child Zone* must be a valid name meeting
+"[Requirements and normalization of domain names in input]".
+
+
+## Intercase dependencies
 
-[INCONSISTENT_DELEGATION]: #outcomes
+None.
 
-[Get-Parent-Zone]: ../Methods.md#method-get-parent-zone
 
+## Terminology
+
+* "Direct Subdomain" - Domain A is considered to be a "direct Subdomain" to
+  domain B if domain A is just the addition of one label at the least significant
+  side, e.g. "foo.domain.com" is a direct subdomain to "domain.com".
+
+* "DNS Lookup" - The term is used when a recursive lookup is used, though
+any changes to the DNS tree introduced by an [undelegated test] must be
+respected.
+
+* "Non-Referral" - See "Referral" above.
+
+* "Referral" - The term refers to a DNS response with [RCODE Name] NoError, AA
```

Comment:
`The term refers to a` -> `A` maybe? It's less verbose but no less clear.

Reply:
Updated.